platform-dom/src/main/java/com/softicar/platform/dom/DomTestMarker.java

```diff
@@ -5,7 +5,33 @@
 public enum DomTestMarker implements IStaticObject {
```

Comment:
shouldn't we sort alphabetically? here and in the other enums

Reply:
already did that, seems I forgot to sort some entries though